src/HealthGPS.Tests/StaticLinearModel.Test.cpp

```diff
@@ -0,0 +1,190 @@
+#include "pch.h"
+
+#include "HealthGPS/static_linear_model.h"
+
+// Create a test fixture for the StaticLinearModel parameters
+class ModelParameters : public::testing::Test {
+    public:
+        hgps::RiskFactorSexAgeTable expected;
+        std::vector<hgps::core::Identifier> names;
+        std::vector<hgps::LinearModelParams> models;
+        std::vector<double> lambda;
+        std::vector<double> stddev;
+        Eigen::Matrix2d correlation;
+        Eigen::MatrixXd cholesky;
+        std::vector<hgps::LinearModelParams> policy_models;
+        std::vector<hgps::core::DoubleInterval> policy_ranges;
+        Eigen::MatrixXd policy_cholesky;
+
+        void SetUp() override {
+            auto expected = hgps::RiskFactorSexAgeTable{};
+            std::vector<double> factorsMale = {200.,40.,30.,2000.,1.,1000.,3.,50.};
+            std::vector<double> factorsFemale = {200.,40.,30.,2000.,1.,1000.,3.,50.};
+
+            std::unordered_map<hgps::core::Identifier, std::vector<double>> factorsMale = {
+                {hgps::core::Identifier{"0"}, factorsMale},
+                {hgps::core::Identifier{"1"}, factorsMale}
```

Comment:
These identifiers will need to be the same as the factors in `names`, so `FoodCarbohydrate`, etc.